src/components/search/SearchResults.tsx

```diff
@@ -20,11 +21,11 @@ const ensureNonNullItems = (result: BaseItemDtoQueryResult) => ({
     Items: result.Items || []
 });
 
-const isMovies = (collectionType: string) => collectionType === 'movies';
+const isMovies = (collectionType: string) => collectionType === CollectionType.Movies;
 
-const isMusic = (collectionType: string) => collectionType === 'music';
+const isMusic = (collectionType: string) => collectionType === CollectionType.Music;
 
-const isTVShows = (collectionType: string) => collectionType === 'tvshows' || collectionType === 'tv';
+const isTVShows = (collectionType: string) => collectionType === CollectionType.Tvshows;
```

Comment:
Is `'tv'` no longer present? I'm assuming that's just a legacy thing.

Reply:
Well i couldn't see tv in the collectiontype enum so assumed it was removed? i can re add it if you reckon it is needed.

Reply:
Probably just a legacy thing... should be fine to drop